byterun/caml/misc.h

```diff
@@ -230,6 +231,7 @@ typedef char char_os;
 #define caml_stat_strdup_to_os caml_stat_strdup
 #define caml_stat_strdup_of_os caml_stat_strdup
 #define caml_copy_string_of_os caml_copy_string
+#define caml_utf8_string_of_os(s) s
```

Comment:
I think convention says the `s` should be `(s)` to ensure that you don't change the meaning of an expression, although I think you'd have to be doing something seriously contrived to manage that with a string pointer.